pandas/tests/io/excel/test_odf.py

```diff
@@ -23,6 +23,13 @@ def test_read_invalid_types_raises():
         pd.read_excel("invalid_value_type.ods")
 
 
+def test_nonexistent_sheetname_raises():
```

Comment:
Thanks for adding! Can you put this in test_readers.py though? Should be consistent across all engines (may need to make a few more changes to get it to work)